### DISCUSSION TITLE
SNMP Exporter for Aruba network Devices

### DISCUSSION THREAD

**Discussion by entekhabasc** [prometheus/snmp_exporter#1257] [Q&A]
How to create snmp.yml file for Aruba Switch in Prometheus, i am using Rocky Linux 9.4

### Host operating system: output of `uname -a`
Rocky Linux 9.4

### snmp_exporter version: output of `snmp_exporter -version`

snmp_exporter, version 0.26.0 

### What device/snmpwalk OID are you using?
HP 5412 switch

**Comment by bastischubert** [comment]
basically get the mibs, look for the OIDs you're interested in and follow the steps outlined in the generator/README.md file
